### ISSUE TITLE
Translation : escaping characters

### ISSUE THREAD

**Issue by j-pillon** [campsych/concerto-platform#29]
Hi,

I am currently translating Concerto in French. I have a problem for escape characters in some cases (especially single quotes).
For the tooltips, I simply add a backslash without problem.
But in the parts titles with a variable write a single quote will always display the HTML code (&#039;) even with backslash.

Example:
TestWizzard.fr_FR.yml (line 7)
```
[...]
form:
    title:
        add: Créer un nouvel assistant de création de tests
        edit: "Editer l'assistant #{0}"
[...]
```
How escape or protect characters in this case ?

**Comment by przemyslawlis** [comment]
Fixed that specific case in: 43f6c0ebe7be6d42f0b023843d2b565875562177
HTML should no longer be escaped there. 
I'm going to leave this issue report open. Please post any other escaping issues you encounter here.

**Comment by przemyslawlis** [comment]
Closing this as there was no response for few days now. Feel free to open new issue report if you find any more problems.
